example-templates/interactive/vscode/repair-port-forward.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -euo pipefail
+
+pattern="kubectl port-forward"
+
+ps_entry="$(ps a | (grep -F "${pattern}" || :) | (grep -v grep || :))"
```

Comment:
This `|| :` thing seems weird - isn't the usual way to mask a failure `|| true`?